news/336.feature

```diff
@@ -0,0 +1 @@
+Provide Containerized workflow
```

Comment:
This should be a `.dev` change, not feature.

Reply:
I was confused between those two. Just as a suggestion, if we label the issues with those things and then update the documentation with `filename should be "{issue-id}.{issue-label}"`. I think this can be much cleaner to do.

Reply:
Good idea, I will create a ticket for it.

Reply:
https://github.com/fedora-infra/the-new-hotness/issues/338